source/_integrations/systemmonitor.markdown

```diff
@@ -65,23 +68,29 @@ All sensors are also marked as diagnostic and won't be automatically added to au
 
 ## Add `process` binary sensor
 
-The `process` binary sensor needs to be configured by the config entry options. Go to **{% my integrations title="Settings > Devices & services" %}**, select the **System Monitor** integration and click **Configure** to select which `process` binary sensors should be created.
+The `process` binary sensor needs to be configured by the config entry options. Go to **{% my integrations title="Settings > Devices & services" %}**, select the **System Monitor** integration and click **Configure**.
```

Comment:
```suggestion
The `process` binary sensor needs to be configured by the config entry options. Go to **{% my integrations title="Settings > Devices & services" %}**, select the **System Monitor** integration and select **Configure**.
```
the style guide prefers [select over click](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/click)